### ISSUE TITLE
Bug: no-invalid-at-rules doesn't recognize margin

### ISSUE THREAD

**Issue by yphoenix** [eslint/css#519]
### Environment

ESLint version: v10.8.0
@eslint/css version: 1.4.0
Node version: v26.5.0
npm version: 11.17.0
Operating System: macOS 26.6

### Which language are you using?

stylesheet

### What did you do?

<details>
<summary>Configuration</summary>

```
Recommended config
```
</details>

```css
@page
{
	margin: 1in 1in 1in 1in;
}
```

### What did you expect to happen?

no error, margin is completely legal css: https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/At-rules/@page

### What actually happened?

```
 3:2  error  Unknown descriptor 'margin' found for at-rule '@page'  css/no-invalid-at-rules
```

[text.css](https://github.com/user-attachments/files/30531773/text.css)

### Link to Minimal Reproducible Example

See above, 

### Participation

- [ ] I am willing to submit a pull request for this issue.

### AI acknowledgment

- [x] I did not use AI to generate this issue report.
- [ ] (If the above is not checked) I have reviewed the AI-generated content before submitting.

### Additional comments

Willing to look at the code and see if I can fix it.

**Comment by DMartens** [comment]
The problem seems to be that our [data](https://github.com/mdn/data/blob/main/css/at-rules.json#L400) from `mdn-data` is out-of-date, as it does not specify `margin`.
So the problem should be fixed in our CSSTree fork.
